projects/1.20/assets/the-dawn-era/dawnera/lang/zh_cn.json

```
@@ -0,0 +1,200 @@
+  "advancements.dawnera.long_live_the_king.description": "遇见霸王龙，一种来自遥远过去的恐怖蜥蜴",
+  "advancements.dawnera.long_live_the_king.title": "国王万岁！",
+  "advancements.dawnera.no_king_rules_forever.description": "将霸王龙从食物链的王座上赶下去",
+  "advancements.dawnera.no_king_rules_forever.title": "再无国王",
+  "advancements.dawnera.psitaco_bell.description": "制作一个鹦鹉嘴龙塔可",
+  "advancements.dawnera.psitaco_bell.title": "鹦鹉塔可钟",
+  "advancements.dawnera.russian_birdwatching.description": "使用望远镜来观察鹦鹉嘴龙",
+  "advancements.dawnera.russian_birdwatching.title": "俄罗斯观鸟",
+  "advancements.dawnera.the_dawn_era.description": "这个时代开始了，你准备好了吗？",
+  "advancements.dawnera.the_dawn_era.title": "破晓纪元",
+  "behaviour.dawnera.aggressive": "aggressive",
+  "behaviour.dawnera.ignore": "ignore",
+  "behaviour.dawnera.passive": "passive",
+  "block.dawnera.alligator_gar_roe": "鳄雀鳝卵",
+  "block.dawnera.ceratosaurus_eggs": "角鼻龙蛋",
+  "block.dawnera.diabloceratops_eggs": "恶魔角龙蛋",
+  "block.dawnera.dimorphodon_eggs": "双形齿龙蛋",
+  "block.dawnera.dunkleosteus_egg": "邓氏鱼蛋",
+  "block.dawnera.psittacosaurus_eggs": "鹦鹉嘴龙蛋",
+  "block.dawnera.sabertooth_salmon_roe": "剑齿鲑卵",
+  "block.dawnera.seagrass_eggs": "附着受精卵的海草",
+  "block.dawnera.struthiomimus_eggs": "似鸵龙蛋",
+  "block.dawnera.sturgeon_roe": "鲟鱼子",
+  "block.dawnera.tall_seagrass_eggs": "附着受精卵的高海草",
+  "block.dawnera.triceratops_egg": "三角龙蛋",
+  "block.dawnera.tyrannosaurus_egg": "霸王龙蛋",
+  "block.dawnera.worm_dirt": "蚯蚓泥土",
+  "death.attack.fracture": "%1$s试图用断肢移动",
+  "eating.dawnera.allow_eat": "allow_eat",
+  "eating.dawnera.deny_eat": "deny_eat",
+  "effect.dawnera.fracture": "撕裂",
+  "entity.dawnera.alligator_gar": "鳄雀鳝",
+  "entity.dawnera.ceratosaurus": "角鼻龙",
+  "entity.dawnera.coelacanth": "空棘鱼",
+  "entity.dawnera.diabloceratops": "恶魔角龙",
+  "entity.dawnera.dimorphodon": "双形齿龙",
+  "entity.dawnera.dunkleosteus": "邓氏鱼",
+  "entity.dawnera.falling_tree": "倒下的树木",
+  "entity.dawnera.mixosaurus": "混鱼龙",
+  "entity.dawnera.pond_snail": "椎实螺",
+  "entity.dawnera.psittacosaurus": "鹦鹉嘴龙",
+  "entity.dawnera.sabertooth_salmon": "剑齿鲑",
+  "entity.dawnera.struthiomimus": "似鸵龙",
+  "entity.dawnera.sturgeon": "鲟鱼",
+  "entity.dawnera.triceratops": "三角龙",
+  "entity.dawnera.tyrannosaurus": "霸王龙",
+  "entity.dawnera.worm": "蚯蚓",
+  "gender.dawnera.female": "female",
+  "gender.dawnera.hermaphrodite": "F&M",
+  "gender.dawnera.male": "male",
+  "gender.dawnera.unknown": "unknown",
+  "gui.dawnera.back": "back",
+  "inspecting.dawnera.gender": "gender",
+  "inspecting.dawnera.health": "health",
+  "inspecting.dawnera.hunger": "hunger",
+  "inspecting.dawnera.name": "name",
+  "inspecting.dawnera.stamina": "stamina",
+  "item.dawnera.alligator_gar": "鳄雀鳝",
+  "item.dawnera.alligator_gar_spawn_egg": "鳄雀鳝刷怪蛋",
+  "item.dawnera.bone_whistle": "骨哨",
+  "item.dawnera.bowl_of_caviar": "鲟鱼鱼子酱",
+  "item.dawnera.bowl_of_sabertooth_salmon_roe": "剑齿鲑鱼子酱",
+  "item.dawnera.bowl_of_sabertooth_salmon_soup": "剑齿鲑汤",
+  "item.dawnera.caviar_sandwich": "鱼子三明治",
```

```suggestion
  "item.dawnera.caviar_sandwich": "鱼子酱三明治",
```
cavinar是鱼子酱，吃生鱼子有点可怕了